Rubberduck.VBEEditor/Events/VBENativeServices.cs

```diff
@@ -165,19 +166,20 @@ private static void OnSelectionChanged(IntPtr hwnd)
             if (pane != null) SelectionChanged?.Invoke(_vbe, new SelectionChangedEventArgs(pane));
         }
 
-        private static string _currentLine;
-        public static event EventHandler<AutoCompleteEventArgs> CaretHidden; // not CodeChanged because wouldn't fire on paste
-        private static void OnCaretHidden(IntPtr hwnd)
+        //private static string _currentLine;
+        public static event EventHandler<AutoCompleteEventArgs> KeyDown; 
+        private static void OnKeyDown(KeyPressEventArgs e)
         {
-            var pane = GetCodePaneFromHwnd(hwnd);
-            if (pane?.Selection.IsSingleCharacter ?? false)
+            using (var pane = GetCodePaneFromHwnd(e.Hwnd))
+            using (var module = pane.CodeModule)
```

Comment:
This does not adhere to the convention for multiple using statements you decided on.

Reply:
*agreed upon :wink: will change it to explicit scopes.